src/main/java/com/google/firebase/database/core/RepoManager.java

```diff
@@ -54,6 +55,8 @@ public static void destroy(Context ctx) {
       instance.destroyInternal(ctx);
     } finally {
       ctx.stop();
+      // TODO(hkj): https://github.com/netty/netty/issues/7310
```

Comment:
Please make this actionable or remove it. (but if you remove the TODO but keep the removeAll() call, please add a comment explaining what it's for)

Reply:
Updated